spec/Checker/Subscription/SubscriptionExpirationCheckerSpec.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace spec\BitBag\SyliusShippingSubscriptionPlugin\Checker\Subscription;
+
+use BitBag\SyliusShippingSubscriptionPlugin\Checker\Subscription\SubscriptionExpirationChecker;
+use BitBag\SyliusShippingSubscriptionPlugin\Entity\SubscriptionAwareInterface;
+use BitBag\SyliusShippingSubscriptionPlugin\Repository\ShippingSubscriptionRepositoryInterface;
+use Doctrine\ORM\NonUniqueResultException;
+use PhpSpec\ObjectBehavior;
+
+class SubscriptionExpirationCheckerSpec extends ObjectBehavior
```

Comment:
I think this test is not needed. Check github.com/BitBagCommerce/BitBagBible , maybe it should be 'final'

Reply:
Removed. All tests are  "final" now.